### PR TITLE
Successful and Unsuccessful category Mod1 test

### PR DESCRIPTION
**Pull request by turnerc1**
## Description

Mes 7976

Created Successful and Unsuccessful tests for mod1 category

* PLEASE INSERT DESCRIPTION

## Checklist

- [x] PR title includes the JIRA ticket number and an understandable description
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket

### CODE REVIEW

File: config/wdio.conf.ts [hunk 1]
```diff
@@ -151,12 +151,12 @@ export const config: WebdriverIO.Config = {
   // reporters: ['spec'],
   reporters: [
     'spec',
-    [video, {
-      saveAllVideos: true, // If true, also saves videos for successful test cases
-      // eslint-disable-next-line max-len
-      videoSlowdownMultiplier: 10, // Higher to get slower videos, lower for faster videos [Value 1-100]
-      videoRenderTimeout: 5, // Max seconds to wait for a video to finish rendering
-    }],
+    // [video, {
```
Comment (IuliaDVSA) [inline config/wdio.conf.ts:154]: Please remove commented code.

**Review by IuliaDVSA — CHANGES_REQUESTED** [review]

File: config/wdio.conf.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import path from 'path';
 import CustomCommand from '../shared/custom/support/lib/addCommands';
 
-const video = require('wdio-video-reporter');
+// const video = require('wdio-video-reporter');
```
[the remark this answers is not in the document]
Comment (JCarterDVSA) [reply config/wdio.conf.ts:4]: he's done this for the pipeline, so the videos don't fill up the disk space.

File: config/wdio.conf.ts (hunk 1 above)
Comment (JCarterDVSA) [reply config/wdio.conf.ts:154]: same here, read above iulia.

File: src/pageobjects/des/office.pageobject.ts [hunk 3]
```diff
@@ -11,7 +11,8 @@ interface OfficePageData {
   showMeQuestion: string;
   weatherCondition: string;
   faultComment: string;
-  eyesightFaultComment: string
+  eyesightFaultComment: string;
+  circuit:'left'| 'right';
 }
 class OfficePageObject extends Page {
   /* eslint-disable no-case-declarations, no-await-in-loop */
```
Comment (JCarterDVSA) [inline src/pageobjects/des/office.pageobject.ts:18]: can probably add this rule to the main eslint file

**Comment by turnerc1** [comment]
> Please up versions

We dont need to up versions as I am not editing the shared repo

File: src/pageobjects/des/office.pageobject.ts (hunk 3 above)
Comment (Cullima) [reply src/pageobjects/des/office.pageobject.ts:18]: agree we should either disable rule globally or refactor

File: config/wdio.conf.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (turnerc1) [reply config/wdio.conf.ts:158]: Added a comment that states to uncomment to enable video reporter

Comment (turnerc1) [reply config/wdio.conf.ts:158]: Added a comment that states to uncomment to enable video reporter

**Review by IuliaDVSA — APPROVED** [review]

File: src/pageobjects/des/office.pageobject.ts (hunk 3 above)
Comment (turnerc1) [reply src/pageobjects/des/office.pageobject.ts]: I have added this rule to the eslint file

**Review by Cullima — APPROVED** [review]

**Pull request merged**